File: Python/11-20/e_11.py
# Escribir una instrucción en que permita crear una lista de números pares del 1 al 10 con una comprensión de lista en python.

numeros_pares = [i for i in range(2, 11, 2)]

# En esta instrucción, range(2, 11, 2) genera una secuencia de números desde 2 hasta 10 (excluyendo 11) con un paso de 2,
# la comprensión de lista [i for i in range(2, 11, 2)] crea una lista que contiene estos números pares.

File: JS/11-20/e_11.js
// Escribir una instrucción en que permita crear una lista de números pares del 1 al 10 con una comprensión de lista en JS.

let numerosPares = [];

for (let i = 2; i <= 10; i += 2) {
    numerosPares.push(i);
}

console.log(numerosPares);

// En JavaScript, no existe una construcción directa para comprensiones de listas como en Python. 
// Puedes utilizar un bucle para lograr el mismo resultado.
// En este ejemplo, se utiliza un bucle for para iterar desde 2 hasta 10 con un incremento de 2 en cada paso,
// se agregan los números pares al array numerosPares. Finalmente, se imprime el array.